controllers/data/views/data.njk

```diff
@@ -1,7 +1,8 @@
+{% extends "layouts/main.njk" %}
 {% from "components/data.njk" import statsGrid %}
 {% from "components/hero.njk" import hero with context %}
 
-{% extends "layouts/main.njk" %}
+{% set copy = __('toplevel.data') %}
```

Comment:
Interesting, I quite like this – `injectCopy()` doesn't allow for the dynamic variable substitution that `__()` does, so perhaps this is the model we should adapt for these sort of pages 👍

Reply:
Yeah and I've also found that the layer of indirection that `injectCopy` adds gets confusing, particularly where `copy` can be overridden later.